src/mvContext.h

```diff
@@ -99,6 +94,8 @@ struct mvIO
 
     // callback registry
     bool manualCallbacks = false;
+
+    unsigned int decimalPoint = '.';  // Equivalent to ImWChar32
```

Comment:
`decimalPoint`: Why don't we declare it as `ImWchar`? (note: **not** `ImWchar32`). That's the way the corresponding field in ImGui is declared. Yes, `set_decimal_point` is commented out at the moment, but eventually we'll uncomment it and will start using this field, so better do it right from the start.

Reply:
Actually I thought that it could have broken something in the parsing step, but if you don't think so I'll do like you said